Frontend/RecRock-Calendar-Frontend/src/app/generated clients/clients.ts

```diff
@@ -0,0 +1,120 @@
+//----------------------
+// <auto-generated>
+//     Generated using the NSwag toolchain v13.15.10.0 (NJsonSchema v10.6.10.0 (Newtonsoft.Json v9.0.0.0)) (http://NSwag.org)
+// </auto-generated>
+//----------------------
+
+/* tslint:disable */
+/* eslint-disable */
+// ReSharper disable InconsistentNaming
+
```

Comment:
as first step manually decorate this calss with 
@[Injectable](https://angular.io/api/core/Injectable)({
  providedIn: 'root',
})
this way the class can be injected in any component

Reply:
see https://angular.io/tutorial/toh-pt4